Prose/ProseLib/Sources/SidebarFeature/RosterReducer.swift

```diff
@@ -124,8 +124,7 @@ extension Reducer where
           )
           return .none
 
-        case .onDisappear, .selection, .addContactButtonTapped, .addGroupButtonTapped,
```

Comment:
Why?

Reply:
Because I had to add three more cases, which made it very long. We already talked about this, I thought you preferred `default`, but this was an error. I won't do it again.